From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 3 Feb 2020 14:46:15 +0100
Subject: info-overview: Use the Vendor logo prominently

This removes GNOME branding in favour of the vendor branding. We don't
make changes to the text information as the distribution/OS is clearly
visible above the GNOME version.

This may be removed when debian will expose the LOGO property in
/etc/os-release (as per upstream commit f39cd5a93)

See https://bugzilla.gnome.org/show_bug.cgi?id=695691
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=989690
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/base-files/+bug/1931582
---
 panels/info-overview/cc-info-overview-panel.c | 5 +++++
 1 file changed, 5 insertions(+)

Index: gnome-control-center/panels/info-overview/cc-info-overview-panel.c
===================================================================
--- gnome-control-center.orig/panels/info-overview/cc-info-overview-panel.c	2022-03-31 05:05:48.379000000 +0000
+++ gnome-control-center/panels/info-overview/cc-info-overview-panel.c	2022-03-31 05:05:48.376000000 +0000
@@ -948,6 +948,7 @@
 static void
 setup_os_logo (CcInfoOverviewPanel *panel)
 {
+#if 0
   GtkIconTheme *icon_theme;
   g_autofree char *logo_name = g_get_os_info ("LOGO");
   g_autoptr(GtkIconPaintable) icon_paintable = NULL;
@@ -975,6 +976,10 @@
                                                    gtk_widget_get_direction (GTK_WIDGET (panel)),
                                                    0);
   gtk_picture_set_paintable (panel->os_logo, GDK_PAINTABLE (icon_paintable));
+#else
+  gtk_image_set_from_file (panel->os_logo, "/usr/share/icons/vendor/scalable/emblems/emblem-vendor.svg");
+  gtk_image_set_pixel_size (panel->os_logo, 128);
+#endif
 }
 
 static void
